elastio-s3-changelog/README.md

```diff
@@ -12,7 +12,7 @@ Then, the Elastio `iscan` job reads those events to perform the scanning of new
 1. First, you need to enable Amazon EventBridge for your S3 buckets by following these instructions:
     [Enabling Amazon EventBridge](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications-eventbridge.html).
 2. You can use
-    [this quick-create link](https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/review?templateURL=https://elastio-artifacts-kskorokhodov-us-east-2.s3.us-east-2.amazonaws.com/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
+    [this quick-create link](https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/v1/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
```

Comment:
> `/contrib/v1`

ideally, this should point to the latest version

Reply:
We agreed with @babinskiy that we'll just overwrite the `v1` version template in the bucket with new updates. We just need to make sure we keep the template backward compatible. If we need to do a breaking change we'll use v2. But yeah, there is the duplication in the docs for the template link. I think it's fine. When we do a breaking change we'll definitely think about docs, but anyway I hope we won't break the API